instrumentation/opentelemetry-instrumentation-aiohttp-client/pyproject.toml

```diff
@@ -35,10 +35,6 @@ dependencies = [
 instruments = [
   "aiohttp ~= 3.0",
```

Comment:
Not related to this pr but `dependencies` needs to be updated to add `opentelemetry-semantic-conventions`